tests/test_index.py

```diff
@@ -582,64 +605,79 @@ def test_file_index_noarch_osx64_1(testing_workdir):
+                "icon_hash": None,
+                "icon_url": None,
+                "identifiers": None,
+                "keywords": None,
+                "license": "BSD",
+                "post_link": False,
+                "pre_link": False,
+                "pre_unlink": False,
+                "recipe_origin": None,
+                "run_exports": {},
+                "source_git_url": None,
+                "source_url": None,
+                "subdirs": ["noarch"],
+                "summary": "A microframework based on Werkzeug, Jinja2 and good intentions",
+                "tags": None,
+                "text_prefix": False,
+                "timestamp": 0,
+                "version": "0.11.1",
             },
-            'flask': {
-                'activate.d': False,
-                'binary_prefix': False,
-                'deactivate.d': False,
-                'description': "Flask is a microframework for Python based on Werkzeug and Jinja2. "
-                               "It's intended for getting started very quickly and was developed with best intentions in mind.",
-                'dev_url': 'https://github.com/mitsuhiko/flask',
-                'doc_source_url': None,
-                'doc_url': 'http://flask.pocoo.org/docs/0.10/',
-                'home': 'http://flask.pocoo.org/',
-                'icon_hash': None,
-                'icon_url': None,
-                'identifiers': None,
-                'keywords': None,
-                'license': 'BSD',
-                'post_link': False,
-                'pre_link': False,
-                'pre_unlink': False,
-                'recipe_origin': None,
-                'run_exports': {},
-                'source_git_url': None,
-                'source_url': None,
-                'subdirs': [
-                    'noarch'
-                ],
-                'summary': 'A microframework based on Werkzeug, Jinja2 and good intentions',
-                'tags': None,
-                'text_prefix': False,
-                'timestamp': 0,
-                'version': '0.11.1'
-            }
         },
         "subdirs": [
             "noarch",
             "osx-64",
-        ]
+        ],
     }
     assert actual_channeldata_json == expected_channeldata_json
 
 
 def _build_test_index(workdir):
-    pkgs = api.build(os.path.join(metadata_dir, "_index_hotfix_pkgs"), croot=workdir)
-    for pkg in pkgs:
-        conda_package_handling.api.transmute(pkg, '.conda')
-    api.update_index(workdir)
 
-    with open(os.path.join(workdir, subdir, 'repodata.json')) as f:
+    # build index_hotfix_pkgs: pkgs =
+    # conda_build.api.build(os.path.join(utils.metadata_dir,
+    # "_index_hotfix_pkgs"), croot=workdir) for pkg in pkgs:
+    #     conda_package_handling.api.transmute(pkg, ".conda")
+    # conda_build.api.update_index(workdir) # not testing update_index in the
+    # _build_test_index() helper
+
+    # workdir may be the same during a single test run?
```

Comment:
I think this comment would be better phrased as a statement of uncertainty rather than a question (more direct that way).

As someone reading this code for the first time, I find questions like these to be a little confusing.